episode0/in_memory_hashtable.go

```diff
@@ -11,7 +11,7 @@ func NewInMemoryHashTable() HashTable {
 	return &inMemoryHashTable{m: make(map[string][]byte)}
 }
 
-func (i *inMemoryHashTable) Get(key string) ([]byte, error) {
+func (i inMemoryHashTable) Get(key string) ([]byte, error) {
 	i.lck.RLock()
```

Comment:
I believe this will copy `lck` so won't provide the mutual exclusion we're looking for in the hash table